src/coreComponents/constitutive/solid/CompressibleSolid.hpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+
+/**
+ * @file CompressibleSolid.hpp
+ */
+
+#ifndef GEOSX_CONSTITUTIVE_SOLID_COMPRESSIBLESOLILD_HPP_
+#define GEOSX_CONSTITUTIVE_SOLID_COMPRESSIBLESOLILD_HPP_
+
+#include "constitutive/solid/CoupledSolid.hpp"
+#include "constitutive/NullModel.hpp"
+
+namespace geosx
+{
+namespace constitutive
+{
+
+/**
+ * @brief Provides kernel-callable constitutive update routines
+ *
+ *
+ * @tparam SOLID_TYPE
+ */
+template< typename PORO_TYPE,
+          typename PERM_TYPE >
+class CompressibleSolidUpdates : public CoupledSolidUpdates< NullModel, PORO_TYPE, PERM_TYPE >
```

Comment:
Is PoreCompressibility a better name?

Reply:
I don't know. I wanted a name that included the fact that it also has a permeability model...